src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -1762,26 +1762,30 @@ private CursorEnumerable<RedisValue> SetScanAsync(RedisKey key, RedisValue patte
 
         public RedisValue[] Sort(RedisKey key, long skip = 0, long take = -1, Order order = Order.Ascending, SortType sortType = SortType.Numeric, RedisValue by = default, RedisValue[]? get = null, CommandFlags flags = CommandFlags.None)
         {
-            var msg = GetSortedSetAddMessage(default(RedisKey), key, skip, take, order, sortType, by, get, flags);
-            return ExecuteSync(msg, ResultProcessor.RedisValueArray, defaultValue: Array.Empty<RedisValue>());
+            var features = GetFeatures(key, flags, out ServerEndPoint? server);
```

Comment:
This can be inside `GetSortMessage` just add an out param there ;) See `GetStringGetWithExpiryMessage` for an example!

Reply:
fair.